TeaDiary/Scripts/App/TeaDisplayer.js

```diff
@@ -30,4 +30,15 @@
         typeEditor.val('');
         notesEditor.val('');
     }
+
+    self.buildTeaFromContainers = (id) => {
```

Comment:
They are not containers. Think Form, or Inputs, or Editor. You already call it EditingFields elsewhere.